src/main/java/io/vlingo/symbio/store/state/inmemory/InMemoryStateStoreActor.java

```diff
@@ -32,20 +34,41 @@
   private final List<Dispatchable<RS>> dispatchables;
   private final Dispatcher dispatcher;
   private final Map<String, Map<String, RS>> store;
+  private final RedispatchControl redispatchControl;
 
   public InMemoryStateStoreActor(final Dispatcher dispatcher) {
+    this(dispatcher, 100L, 1000L);
+  }
+
+  public InMemoryStateStoreActor(final Dispatcher dispatcher, long checkConfirmationExpirationInterval, final long confirmationExpiration) {
     if (dispatcher == null) {
       throw new IllegalArgumentException("Dispatcher must not be null.");
     }
     this.dispatcher = dispatcher;
-
     this.adapterAssistant = new StateStoreAdapterAssistant();
     this.store = new HashMap<>();
     this.dispatchables = new ArrayList<>();
+    
+    final DispatcherControl control = selfAs(DispatcherControl.class);
```

Comment:
The `InMemoryStateStoreActor` is its own `DispatcherControl` which works best in this case since all the data is maintained internally (share nothing). I assume that for database backed implementations the `DispatcherControl` would be a separate object. Note that it would not be required to be an actor since it could be performing the unconfirmed queries in the thread provided by the delivery of `intervalSignal()` to `RedispatchControlActor`. Please confirm.